+sig/timeplot.m

```diff
@@ -38,6 +38,7 @@
 % TODO Vararg for axes name value args, e.g. LineWidth
 % TODO Deal with logging signals & registries & subscriptable signals
 % TODO Deal with strings, arrays, structures, cell arrays
+% TODO Prettify labels/titles (create listeners for them to reset position each time a signal is updated?)
```

Comment:
I don't like the current position of the titles as they get in the way of visualizing the plots, but don't have a better place for them currently, as if you just set them to a new position they stay at the position permanently, but since the plot updates you would need to have the positions update in accordance with the plot